src/test/java/org/kainos/ea/integration/JobRoleIntegrationTest.java

```diff
@@ -26,6 +27,16 @@ void getJobRolesShouldReturnListOfJobRoles() {
                 .request()
                 .get(List.class);
 
+        // Parse object.
+        ObjectMapper mapper = new ObjectMapper();
+        List<JobRole> parsedResponse = mapper.convertValue(response, new TypeReference<List<JobRole>>() {});
+
+        for (int i = 0; i < parsedResponse.size(); i++) {
+            Assertions.assertNotNull(parsedResponse.get(i).getCapability());
+        }
+        for (int i = 0; i < parsedResponse.size(); i++) {
+            Assertions.assertNotNull(parsedResponse.get(i).getBandLevel());
+        }
```

Comment:
you only need 1 loop to check both of these, otherwise you're looping through the same list 2 times